### PR TITLE
GF 4-6

### PR DESCRIPTION
**Pull request by maijapapaya**
@jaredh159 @Henderjay

### CODE REVIEW

File: epistles-v1/original/04-epistles.adoc [hunk 1]
```diff
@@ -548,8 +552,8 @@ G+++.+++ F.
 [.blurb]
 ==== To Friends in the North.
 
-All Friends in Westmoreland, Cumberland, Bishopric, New Castle-side, Northumberland,
-and Lancashire,
+[.blurb]
+=== All Friends in Westmoreland, Cumberland, Bishopric, New Castle-side, Northumberland, and Lancashire,
```
Comment (maijapapaya) [inline epistles-v1/original/04-epistles.adoc]: Should this be a blurb? I didn't know what to do with it. It didn't look right as a salutation.

File: epistles-v1/original/06-epistles.adoc [hunk 2]
```diff
@@ -796,6 +803,7 @@ and condemneth the contrary.
 In which power the Lord God Almighty preserve you to his glory.
 Amen.
 
+[.offset]
 Live in the life of God, and feel it.
```
Comment (maijapapaya) [inline epistles-v1/original/06-epistles.adoc:811]: I'm not sure if this should be offset.

File: epistles-v1/original/04-epistles.adoc (hunk 1 above)
Comment (jaredh159) [reply epistles-v1/original/04-epistles.adoc]: yeah, i tried it a couple ways too... i didn't really love any of them, but I think i lean towards making this a _salutation_ instead.

File: epistles-v1/original/04-epistles.adoc [hunk 3]
```diff
@@ -1182,6 +1186,7 @@ and lose your crown; which none receives, but he that continues to the end.
 [.signed-section-signature]
 G+++.+++ F.
 
+[.signed-section-context-close]
```
Comment (jaredh159) [inline epistles-v1/original/04-epistles.adoc]: can you switch this to be a _postscript_?

File: epistles-v1/original/05-epistles.adoc [hunk 4]
```diff
@@ -1189,4 +1190,5 @@ in the still life, patient, to the answering the good in all,
 to the refreshing one of another, and not to the stumbling.
 But mind that which keepeth in unity, in the life, though never so little.
 
+[.signed-section-signature]
```
Comment (jaredh159) [inline epistles-v1/original/05-epistles.adoc:1203]: No problem with this line, but can you make lines 606-611 of this file (ch. 5) a _postscript_ while you’re at it? -- also line 981 - 983 ("And Friends, in the measure of life,...") -- basically any time you see him sign an epistle _twice_ the chunk after the first signature should be a PS. 👍

File: epistles-v1/original/06-epistles.adoc [hunk 5]
```diff
@@ -564,6 +570,7 @@ Amen.
 [.signed-section-signature]
 G+++.+++ F.
 
+[.signed-section-context-close]
```
Comment (jaredh159) [inline epistles-v1/original/06-epistles.adoc]: I think this would be better as a PS.

File: epistles-v1/original/06-epistles.adoc (hunk 2 above)
Comment (jaredh159) [reply epistles-v1/original/06-epistles.adoc:811]: I think it's good. 👍

**Review by jaredh159 — CHANGES_REQUESTED** [review]
Thanks Maija, just a couple changes needed here, mostly postscript stuff.  👍

**Comment by jaredh159** [comment]
All looks good, I quick fixed the lint error, and am merging.

By the way, I did some work this week updating the pull-request linter, so it should not give any more spurious errors. Which means, going forward, if you see a lint error on a PR, you can pay attention to it and fix it, when possible. Thanks!

**Review by jaredh159 — APPROVED** [review]

**Pull request merged**